src/main/python/core/config_reader.py

```diff
@@ -19,6 +20,7 @@ def parse(self):
             self._check_all_sections_present()
             self.defaults = self.config[Sections.DEFAULTS.value]
             self.overwrites = self.config[Sections.OVERWRITES.value]
+            self.datafiles = self.config[Sections.DATAFILES.value]
```

Comment:
Comment on L28:
The path to the configuration file is currently hardcoded and located within the project directory. This makes it hard to configure CAPICE when running from a e.g. EasyBuild module or Singularity container. I suggest to rename config.cfg to default.cfg and allow users to pass their own configuration file through the command-line interface with updated configuration values.

Reply:
Fixed in #54